utopia-remix/app/services/fgaClient.server.ts

```diff
@@ -0,0 +1,65 @@
+import { CredentialsMethod, ErrorCode, FgaApiValidationError, OpenFgaClient } from '@openfga/sdk'
+import { ServerEnvironment } from '../env.server'
+
+const mockOpenFgaClient = {
+  write: async () => {},
+  check: async () => ({ allowed: true }),
+}
+
+export const fgaClient = createFgaClient()
```

Comment:
I think this would be a great place to reuse the `singleton` helper available in `singleton.server.ts`